debezium-connector-jdbc/src/test/java/io/debezium/connector/jdbc/JdbcSinkConnectorConfigTest.java

```diff
@@ -171,46 +171,68 @@ public void testDeprecatedDatabaseTimeZone() {
     @Test
     public void testDeprecatedTableNamingStrategy() {
         var properties = Map.of(
-                JdbcSinkConnectorConfig.DEPRECATED_TABLE_NAMING_STRATEGY, DefaultTableNamingStrategy.class.getName(),
-                JdbcSinkConnectorConfig.DEPRECATED_TABLE_NAME_FORMAT, "kafkadepdep_${topic}");
+                JdbcSinkConnectorConfig.DEPRECATED_TABLE_NAMING_STRATEGY, "table.naming.strategy",
+                JdbcSinkConnectorConfig.COLLECTION_NAMING_STRATEGY, JdbcSinkConnectorConfig.DEPRECATED_TABLE_NAMING_STRATEGY);
+
         final JdbcSinkConnectorConfig config = new JdbcSinkConnectorConfig(properties);
 
+        Field.Set fields = Field.setOf(JdbcSinkConnectorConfig.COLLECTION_NAMING_STRATEGY_FIELD);
+
         AtomicReference<String> errorMessage = new AtomicReference<>();
-        assertThat(config.validateAndRecord(List.of(JdbcSinkConnectorConfig.COLLECTION_NAMING_STRATEGY_FIELD), errorMessage::set)).isTrue();
+        boolean isValid = config.validateAndRecord(fields, errorMessage::set);
+        assertTrue(isValid, "Validation should pass.");
         assertEquals(
-                "The 'collection.naming.strategy' value is invalid: Warning: Using deprecated config option \"table.naming.strategy\".",
-                errorMessage.get());
-        assertThat(config.validateAndRecord(List.of(JdbcSinkConnectorConfig.COLLECTION_NAME_FORMAT_FIELD), errorMessage::set)).isTrue();
-        assertEquals("The 'collection.name.format' value is invalid: Warning: Using deprecated config option \"table.name.format\".",
+                "The 'collection.naming.strategy' value 'table.naming.strategy' is invalid: Warning: Using deprecated config option \"table.naming.strategy\".",
                 errorMessage.get());
+    }
+
+    @Test
+    public void testProxyWithDeprecatedTableNamingStrategy() {
```

Comment:
Since we have moved things to the transform, I suspect a lot of the changes in this class can be removed, too.